### PR TITLE
[WFLY-15375] Fix unstable test

### PR DESCRIPTION
**Pull request by jasondlee**
https://issues.redhat.com/browse/WFLY-15375

Increase ryuk timeout
Modify assume() to check for Docker in setup
Remove "always pull" image policy
Move test container setup to separate class

### CODE REVIEW

File: testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java [hunk 1]
```diff
@@ -187,12 +187,13 @@
...
-        // to the internal mpExtensions Set defined on this class.
-        CURRENT(MAJOR, WILDFLY_24_0, Arrays.asList(
+        WILDFLY_25_0("WildFly25.0", WILDFLY_24_0, Arrays.asList(
                 "org.wildfly.extension.elytron-oidc-client",
                 "org.wildfly.extension.opentelemetry"
-        ));
+        )),
+        // If an extension is added to this enum, also check if it is supplied only by wildfly-galleon-pack. If so, add it also
+        // to the internal mpExtensions Set defined on this class.
+        CURRENT(MAJOR, WILDFLY_25_0);
```
Comment (bstansberry) [inline testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java:196]: this commit can be dropped now; it's in main

**Comment by bstansberry** [comment]
Nit: 

s/Fix unstable text/Fix unstable test/g 

;)

File: testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java (hunk 1 above)
Comment (jasondlee) [reply testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java]: Updated

**Comment by jasondlee** [comment]
> Nit:
> 
> s/Fix unstable text/Fix unstable test/g
> 
> ;)

Just noticed that as well. Fixed in the commit. :)

**Review by bstansberry — APPROVED** [review]

**Pull request merged**